README.md

```diff
@@ -50,8 +50,6 @@ Local development is normally done inside docker-compose so that the supporting
 Make docker-compose start the supporting services
 
     docker-compose run --rm web python wait_for_postgres.py
-    docker-compose run --rm web python manage.py collectstatic --no-input
-    
```

Comment:
Does it mean this is run "somewhere" else or why don't we need it documented?

Reply:
This is part of the local run configuration that I'll be re-documenting. As for now, we don't need to actually run this command as we don't have static resources really, so it works fine without it.